spec/cb/clients/job_spec.rb

```diff
@@ -12,108 +12,6 @@
 
 module Cb
   describe Cb::Clients::Job do
-    describe '#search' do
```

Comment:
Yeah, this method should be removed as it was then entry point for the old v1 job search.  It won't work with the new OAuth based requests of job search v3.